ballerina-shell/modules/shell-cli/src/test/java/io/ballerina/shell/cli/test/validator/ValidatorTest.java

```diff
@@ -40,7 +40,7 @@ public void testValidator() {
         Assert.assertTrue(inputValidator.isComplete("int n = flag ? 1 : 2"));
 
         Assert.assertFalse(inputValidator.isComplete("int[] x = [1,2,3,"));
-        Assert.assertFalse(inputValidator.isComplete("map<int> m = {"));
+//        Assert.assertFalse(inputValidator.isComplete("map<int> m = {"));
```

Comment:
Can you mentioned the issue which causes to comment this test?

Reply:
We can enable this test after fixing the https://github.com/ballerina-platform/ballerina-lang/issues/36385.

Reply:
Yes, what I mean was putting a comment mentioning this issue number

Reply:
Done.